packages/passport-interface/src/RequestTypes.ts

```diff
@@ -158,3 +151,39 @@ export interface FeedRequest<T extends PCDPackage = PCDPackage> {
 export interface FeedResponse {
   actions: PCDAction[];
 }
+
+export interface PCDpassUserJson {
+  superuserEventConfigIds: string[];
+  uuid: string;
+  commitment: string;
+  email: string;
+  salt: string | null;
+}
+
+export interface ZupassUserJson {
+  email: string;
+  name: string;
+  role: ZuzaluUserRole;
+  visitor_date_ranges?: DateRange[] | null;
+  uuid: string;
+  commitment: string;
+  salt: null; // Zupass users never have a password salt
+}
+
+export interface ConfirmationEmailResponse {
+  token: string;
+}
+
+export interface SaltResponse {
+  salt: string | "";
```

Comment:
What does this type accomplish?  I'd expect empty string to be a strict subset of `string`.

Reply:
no good reason - will fix

Reply:
fixed